frontend/src/components/CustomTable.tsx

```diff
@@ -383,7 +396,14 @@ export default class CustomTable extends React.Component<CustomTableProps, Custo
       request.sortBy += ' desc';
     }
 
-    return this.props.reload(request);
+    let result = '';
+    this.setStateSafe({ isBusy: true });
```

Comment:
Can the `setStateSafe` a few lines above be merged here?

Reply:
The reverse is probably safer. Done.